libraries/chain/include/graphene/chain/exceptions.hpp

```diff
@@ -85,7 +81,6 @@ namespace graphene { namespace chain {
    FC_DECLARE_EXCEPTION( chain_exception, 3000000, "blockchain exception" )
```

Comment:
My thought was that exceptions are an implementation detail in general, not part of the protocol itself. If some third party implementation wanted to support our protocol, it might use our protocol library but use different error handling entirely.

Reply:
Makes sense.